### DISCUSSION TITLE
Have Dragon Egg Respawn

### DISCUSSION THREAD

**Discussion by FoxWithSox** [Zrips/CMI#5945] [Feature Requests]
A config option to be able to respawn the dragon egg again

**Comment by CloudeLecaw** [comment]
what you mean exactly? i can simply spawn you a dragonegg and with new servers you also can simply respawn the dragon as often as you want with crafting 4 endcrstals.

What you wanne do? let your players farm xxxx dragoneggs?
Or simply a command that let a new egg drops if the next spawned enderdragon is killed?

You also can use a CustomDrope plugin and grand the DragonEgg as a drope or with a Dropchance - plugins like [DropEdit2](https://www.spigotmc.org/resources/%E2%9C%85-50-off-dropedit-2-advanced-drop-editor-%E2%9C%8D%EF%B8%8F.60331/) can manage this well.

**Comment by FoxWithSox** [comment]
Yes sorry for not specifying, I mean something to let a new egg drop each time the dragon is killed.

**Comment by CloudeLecaw** [comment]
Hm DropEdit2.. define a chance of 100% for a dragonegg in it ;)
Then kill the first Dragon by youself because the First would drop the vanilla egg + the additional one.

Its a good plugin and may a faster solution because i dont think zrips wanne add aditional drops for now.
